tests/models/test_fallback.py

```diff
@@ -277,6 +277,7 @@ def test_all_failed_instrumented(capfire: CaptureLogfire) -> None:
                     'logfire.span_type': 'span',
                     'logfire.msg': 'chat fallback:function:failure_response:,function:failure_response:',
                     'logfire.level_num': 17,
+                    'gen_ai.response.model': 'fallback:function:failure_response:,function:failure_response:',
```

Comment:
This line is easy to account for in other ways, this is not enough reason to restrict the version of logfire.

Reply:
These tests won't pass as they are. You can make them pass.

It's good to test the newest version of logfire, this PR is a good idea, but the minimum version doesn't need to be raised.